ui/base/src/main/kotlin/org/cru/godtools/base/ui/util/ModelUtils.kt

```diff
@@ -34,4 +34,6 @@ private fun Context.getToolCategoryStringRes(category: String) =
         0 -> null
         else -> resources.getString(id)
     }
+fun String.getLocalStringFromToolCategory(context: Context) =
+    context.getToolCategoryStringRes(this) ?: this.replace("_", " ")
```

Comment:
which shouldn't be a problem because the first thing that method does is just read the category out of the tool and then generates the name exclusively from that category

Reply:
it'll probably look something like this:
```kotlin
fun Tool?.getCategory(context: Context, locale: Locale? = null) = getToolCategoryName(this?.category, context, locale)

fun getToolCategoryName(category: String?, context: Context, locale: Locale? = null)
```